src/foam/nanos/cron/Schedulable.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'Schedulable',
+  mixins: [
+    'foam.nanos.auth.CreatedAwareMixin',
+    'foam.nanos.auth.CreatedByAwareMixin'
+  ],
+
+  implements: [
+    'foam.nanos.auth.ServiceProviderAware',
+  ],
+
+  properties: [
+    {
```

Comment:
Please add a String id property.

Reply:
And an nspec for the schedulableDAO

Reply:
Added nSpec and just extended Cron so don't need to add id.